ppx_view/lib/deconstructor.ml

```diff
@@ -1,25 +1,15 @@
 open Ppx_ast.V4_07
 
-let expression ~loc expr =
-  match Expression.to_concrete expr with
-  | None -> Error.conversion_failed ~loc "expression"
-  | Some {pexp_loc; pexp_desc; _} ->
-    match Expression_desc.to_concrete pexp_desc with
-    | None -> Error.conversion_failed ~loc:pexp_loc "expression_desc"
-    | Some desc -> (pexp_loc, desc)
+let expression expr =
```

Comment:
I think we might even remove that module now. The lack of a exception based `to_concrete` function was the reason why I added it in the first place. I'll look into doing that when improving error reporting in ppx_view!